client/src/components/lower-info-area/new-and-better-way.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div id="new-and-better-way" class="pt-16">
+  <div id="new-and-better-way" class="pt-16 pb-4">
```

Comment:
I have found that a best practise is to add top padding (or margin) to the element below this one -- this prevents THIS element from "assuming" how much whitespace is required below it. 

(Whereas the element below knows how much whitespace to have above it.)

Can this be fixed here?

Reply:
It wont work here since the background color of this div and the div below is different

Reply:
OK, you found the exception to my rule :)

Reply:
(Although to be technical, it's not whitespace if it's a different colour ;) )